NEW_COMPONENT.md

```diff
@@ -51,17 +51,17 @@ Create an `exports` array in the poster module and add the poster component, lik
 
 ## Step4: Change selector to fd prefix
 
-The Fundamental NGX library uses 'fd' as the component and directive prefix. Open `poster.component.ts` and change the component's 'app' prefix to 'fd', like so:
+The Fundamental Library for Angular uses 'fd' as the component and directive prefix. Open `poster.component.ts` and change the component's 'app' prefix to 'fd', like so:
 
 ```TypeScript
   selector: 'fd-poster',
 ```
 
 ## Step5: Add poster module to list of imports (can be in FundamentalNgxCore)
 
-The documentation application is importing every component in the fundamental-ngx library module. Open `fundamental-ngx.module.ts` and add `import { PosterModule } from './poster/poster.module';` to the list of imports at the top of the file, then add `PosterModule` to the array of exports.
+The documentation application is importing every component in the Fundamental Library for Angular module. Open `Fundamental Library for Angular.module.ts` and add `import { PosterModule } from './poster/poster.module';` to the list of imports at the top of the file, then add `PosterModule` to the array of exports.
 
-We must also add `export * from './lib/poster/poster.module';` to the `fundamental-ngx/src/public_api.ts` file. The <fd-poster> component will be an exported member of the fundamental-ngx module as well as the poster module.
```

Comment:
same here `fundamental-ngx module` is a file name

Reply:
done